src/app/views/pages/clientes/clientes.component.html

```diff
@@ -183,9 +188,26 @@ <h5 class="modal-title" id="modalActualizar">Actualizar</h5>
                 </div>
                 <div class="modal-footer">
                     <button type="button" class="btn btn-secondary" (click)="modal.close('by: close button')">Cerrar</button>
-                    <button type="submit" class="btn btn-primary" [disabled]="actualizarCliente.invalid">Guardar</button>
+                    <button type="submit" class="btn btn-primary" [disabled]="actualizarCliente.invalid">Editar</button>
                 </div>
             </form>
         </div>
     </div>
+</ng-template>
```

Comment:
excelente el modal de eliminar, pero pasa lo siguiente o lo haces componente para poderlo reutilizar por todo el proyecto o implementamos algo como esto https://sweetalert2.github.io/ (si lo instalas busca como implementarlo en angular) te dejo algunas referencias https://www.c-sharpcorner.com/article/how-to-use-sweetalert-in-angular10/